README.md

```diff
@@ -365,6 +366,34 @@ The maxLength attribute added to the input. Specifies the maximum number of char
     ...>
 ```
 
+<a name="inline"></a>
+##### inline (optional)
+The inline attribute use to determine that selected tags and input field will be put in the same div block or not. If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute.
```

Comment:
The inline attribute is deciding whether to render the  input field in same div as selected tags or the sibling div. So I think we can remove this line
`If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute
`